src/components/common/YourCompany/YourCompany.vue

```diff
@@ -300,6 +300,17 @@
         />
       </div>
     </template>
+
+    <!-- Business Start Date (changes, conversions and firm corrections only) -->
+    <template v-if="isBenCorrectionFiling">
+      <v-divider class="mx-4 my-1" />
+
+      <ResolutionDate
+        class="section-container"
+        :class="{'invalid-section': invalidResolutionDate}"
+        :invalidSection="invalidResolutionDate"
+      />
+    </template>
```

Comment:
This made sense when this was thought to be Special Resolution Date.

However, as we now believe this is the Articles section (still to be confirmed in ticket) then it will be new block in BenCorrection.vue.